test/e2e/network/networking.go

```diff
@@ -191,6 +191,36 @@ var _ = SIGDescribe("Networking", func() {
 			}
 		})
 
+		ginkgo.It("should function for pod-Service(hostNetwork): http", func() {
+			config := e2enetwork.NewNetworkingTestConfig(f, e2enetwork.EndpointsUseHostNetwork(), e2enetwork.PodHTTPPort(8090), e2enetwork.PodUDPPort(8091))
```

Comment:
I still feel like this is more likely to lead to conflicts/bugs in the future than just having `NewNetworkingTestConfig` pick ports itself. Like, if someone copies+pastes the test, they have to remember to change the ports, and also scan through all the other tests to figure out what the next available port is.

Reply:
Maybe I'm not understanding well your proposal, do you propose to have a pool of ports and then the tests allocate ports from it?
you need to size it correctly so you never have to change it, because adding a new test will imply to count all the number of tests using the pool, you can not oversubscribe. What size should we reserve, 100 ports? 8900 - 9000, 
Another problem I see is, if someone adds a new test that uses a port belonging to the pool range, i.e 8915, you can not grep in the code, and if you don't know that there are some tests in the code that allocate from a pool, you'll be having port conflict errors with different tests.

I still think constants are a better option, :smile:

Reply:
> Another problem I see is, if someone adds a new test that uses a port belonging to the pool range, i.e 8915, you can not grep in the code

True. OK.

(Though really, the way the code is written, we could just let the apiserver allocate free ports itself and just read them back and set `config.NodeHTTPPort` etc from that...)

Reply:
actually, I'm not happy with my solution too, let' s see if we can find a middle ground ...